Kickstarter-iOS/Views/RewardCardView.swift

```diff
@@ -120,7 +140,7 @@ public final class RewardCardView: UIView {
     self.minimumPriceConversionLabel.rac.hidden = self.viewModel.outputs.conversionLabelHidden
     self.minimumPriceConversionLabel.rac.text = self.viewModel.outputs.conversionLabelText
     self.descriptionLabel.rac.text = self.viewModel.outputs.descriptionLabelText
-    self.estimatedDeliveryDateLabel.rac.hidden = self.viewModel.outputs.estimatedDeliveryDateLabelHidden
+    self.estimatedDeliveryStackView.rac.hidden = self.viewModel.outputs.estimatedDeliveryDateLabelHidden
```

Comment:
Not such a big deal in this case but typically we'd want to keep the output name matching the element so in this case we'd rename to `estimatedDeliveryStackViewHidden` but it's still pretty clear here so just mentioning.

Reply:
Renamed and pushing it up. Will merge when that finishes on CI.